src/utils/browserCheck.ts

```diff
@@ -10,6 +10,13 @@ import { t } from '@nextcloud/l10n'
 
 const parser = new UAParser()
 const browser = parser.getBrowser()
+const os = parser.getOS()
+
+/**
+ * Per-OS flags
+ */
+
+export const isMac = os.name === 'Mac OS'
```

Comment:
Shouldn't we update in the meantime?
https://docs.uaparser.dev/intro/whats-new.html#version-2-0

Reply:
We can

Reply:
Then let's follow-up, if you don't want to mix PRs